integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/common/BridgeUtils.java

```diff
@@ -50,19 +51,26 @@ private static String getAccessToken() {
                 .getToken();
     }
 
-    public static String getOrRetrieveBridgeEndpoint(TestContext context) {
-        if (context.getEndPoint() == null) {
+    public static String getOrRetrieveBridgeEndpoint(TestContext context, String testBridgeName) {
+        return getOrRetrieveBridgeEndpoint(context, testBridgeName, false);
+    }
+
+    public static String getOrRetrieveBridgeEndpoint(TestContext context, String testBridgeName,
+            boolean includeRemovedBridges) {
```

Comment:
Is it needed to distinguish `includeRemovedBridges`? Wouldn't it be simpler to track removed bridges in the same way as existing bridges? Test scenarios should be aware of bridge status, IMHO there is no need to track it also in code.

Reply:
This is over-engineering :D will remove that

Reply:
done